src/pages/PingDetail.svelte

```diff
@@ -95,7 +95,7 @@
           </a>
         </td>
       </tr>
-      {#if selectedAppVariant.looker_url}
+      {#if selectedAppVariant.looker_explore}
```

Comment:
`selectedAppVariant.looker_explore` is always true because it's an object.

```suggestion
      {#if selectedAppVariant.looker_explore.url}
```

Reply:
Should be resolved with the other suggested change.